src/browser/dom/attribute.zig

```diff
@@ -25,24 +25,24 @@ pub const Attr = struct {
     pub const prototype = *Node;
     pub const subtype = .node;
 
-    pub fn get_namespaceURI(self: *parser.Attribute) !?[]const u8 {
-        return try parser.nodeGetNamespace(parser.attributeToNode(self));
+    pub fn get_namespaceURI(self: *parser.Attribute) ?[]const u8 {
+        return parser.nodeGetNamespace(parser.attributeToNode(self));
     }
 
-    pub fn get_prefix(self: *parser.Attribute) !?[]const u8 {
-        return try parser.nodeGetPrefix(parser.attributeToNode(self));
+    pub fn get_prefix(self: *parser.Attribute) ?[]const u8 {
+        return parser.nodeGetPrefix(parser.attributeToNode(self));
     }
 
     pub fn get_localName(self: *parser.Attribute) ![]const u8 {
```

Comment:
you don't remove the error return here on purpose? (same for the 2 following)

Reply:
The underlying netsurf function still returns an error, I just removed the redundant `try`.